tests/system/Database/Builder/WhenTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Database\Builder;
+
+use CodeIgniter\Test\CIUnitTestCase;
+use CodeIgniter\Test\Mock\MockConnection;
+
+/**
+ * @internal
+ */
+final class WhenTest extends CIUnitTestCase
+{
+    /**
+     * @var MockConnection
+     */
+    protected $db;
```

Comment:
Use property type, instead of `@var`.

Reply:
@kenjis I'm not sure exactly what you're looking for here....